src/ExtensionsSample/Program.cs

```diff
@@ -35,7 +35,7 @@ public static void Main(string[] args)
             config.UseFiles(filesConfig);
             config.UseTimers();
             config.UseSample();
-            config.UseEasyTables();
+            config.UseMobileTables();
```

Comment:
Renaming seems like a decent idea to me. Maybe .MobileApps? @paulbatum  -- any preference? This would affect the project name, project, and namespaces.